caffe2/opt/optimize_ideep.cc

```diff
@@ -3,6 +3,7 @@
 #include "caffe2/opt/fusion.h"
 
 #ifdef CAFFE2_USE_MKLDNN
+#include <cpuinfo.h>
```

Comment:
Where does this come from?

Reply:
the src of cpuinfo can be found under third_party/cpuinfo.
it is static built in to caffe2 lib by default.
it is introduced here to check if the avx512 is supported on current machine.
because the conv + sum fusion is only available on avx512 path so far.